### PR TITLE
Ngrok support has been added

### PR DESCRIPTION
**Pull request by sankethj**
Added auth-token configuration for pyngrok which is compulsory now to use Ngrok. Need to add our own Ngrok auth-token in constraint.py file

### CODE REVIEW

**Comment by ARanonymous** [comment]
Ngrok doesn't always work, can it be replaced by cloudflare?

**Comment by karma9874** [comment]
True, ngrok has some issues checking for alternatives

**Comment by ARanonymous** [comment]
I switched to cloudflare it's perfect u should try it

**Comment by ARanonymous** [comment]
I use cloudflare with other tools I haven't tried it yet with this one but there are alternatives to ngrok that are being used for this tool i saw someone on youtube using portmap.io I'll leave a link to the vid.

https://youtu.be/-fWqPKZiWPw

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ git clone https://github.com/karma9874/AndroRAT.git
 cd AndroRAT
 pip install -r requirements.txt
 ```
+* Add your own ngrok auth-token in **constraints.py**
```
Comment (Srivaliraj) [inline README.md:37]: yeah

File: constraints.py [hunk 2]
```diff
@@ -0,0 +1 @@
+authtoken="xxxxxxxxxxxxx"
```
Comment (Srivaliraj) [inline constraints.py:1]: what is this

**Pull request closed without merging**